packages/webapp/scripts/build.js

```diff
@@ -89,6 +89,8 @@ async function developmentBuild() {
 					? 'react/cjs/react.development.js'
 					: name === 'react-dom'
 					? 'react-dom/cjs/react-dom.development.js'
+					: name === 'ecsy'
```

Comment:
This is required for ecsy but not for PIXI. Is there any way we can auto-detect which modules will need this, and where the necessary file is, instead of having to manually add them here?

Reply:
Mmm, Similar to node resolve plugin we have to resolve the appropriate modules here. So first preference for esm, then cjs then something else. Not entirely sure what we can do. Will look into this.